package.json

```diff
@@ -20,12 +20,12 @@
   },
```

Comment:
I would think about using [npm shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap) as it can save you pains in the future with different package versions being installed on other devs or build systems.